pretty_midi/pretty_midi.py

```diff
@@ -504,13 +571,38 @@ def get_chroma(self, fs=100, times=None):
                 Chromagram of MIDI data, flattened across instruments
         '''
         # First, get the piano roll
-        piano_roll = self.get_piano_roll(fs=fs, times=times)
+        piano_roll = self.get_piano_roll(
+            fs=fs,
+            times=times,
+            include_pitch_bends=include_pitch_bends)
         # Fold into one octave
         chroma_matrix = np.zeros((12, piano_roll.shape[1]))
         for note in range(12):
             chroma_matrix[note, :] = np.sum(piano_roll[note::12], axis=0)
         return chroma_matrix
 
+    def get_key_signatures(self):
```

Comment:
If __key_changes and __time_signatures are public, we don't need these getter functions.